v2/component_analyses.yaml

```diff
@@ -0,0 +1,169 @@
+                            cvss: "1.3"
+                      recommended_version : 2
+                      message: "3 vulnerabilities within this dependency, 0 exploitable vulnerabilities"
+                      severity: "high"
+                      public_vulnerabilities_count: 2
+                      private_vulnerabilities_count: 1
+                      exploitable_vulnerabilities_count: 0
+                      vendor_package_link: "https://snyk.io/vuln/maven:org.zeroturnaround%3Azt-zip"
+                FreeUser:
+                  value:
+                    recommendation:
+                      component_analyses:
+                        cve:
+                          - id: "CVE-2018-1002201"
+                            cvss: "1.2"
+
+                          - id: "CVE-2018-1002202"
+                            cvss: "1.3"
+                      recommended_version : 2
+                      message: "2 vulnerabilities within this dependency"
+                      severity: "high"
+                      public_vulnerabilities_count: 2
+                      vendor_package_link: "https://snyk.io/vuln/maven:org.zeroturnaround%3Azt-zip"
+
+        202:
+          description: Package is currently Unavailable. Please try again later
+          content: {}
+        401:
+          description: Request unauthorized
+          content: {}
+        404:
+          description: Data not found
+          content: {}
+components:
+  schemas:
+    VulnerabilityDetails:
+      title: Vulnerability Details
+      type: object
+      properties:
+        vulneability:
+          type: array
+          items:
+            type: object
+            properties:
+              id:
+                type: string
+              cvss:
+                type: string
+      description: Information of the vulnerability found
+
+    RegisteredUser:
+      title: Vendor Private Recommendation Information
+      type: object
+      properties:
+        recommendation:
+          type: object
+          properties:
+            recommended_version:
+              type: integer
+            affected_version:
+              type: string
+            component_analyses:
+              $ref: '#/components/schemas/VulnerabilityDetails'
+            message:
+              type: string
+            severity:
+              type: string
+            public_vulnerabilities_count:
+              type: integer
+            private_vulnerabilities_count:
+              type: integer
+            exploitable_vulnerabilities_count:
+              type: integer
+            vendor_package_link:
+              type: string
+      description: Vendor Recommendation information for Premium Recommendation.
+
+    FreeUser:
+      title: Vendor Public Recommendation Information
+      type: object
+      properties:
+        recommendation:
+          type: object
+          properties:
+            recommended_version:
+              type: integer
```

Comment:
recommended_version type must be a string

Reply:
Ack!